DESCRIPTION

```diff
@@ -1,23 +1,27 @@
 Package: diseasy
-Title: What the Package Does (One Line, Title Case)
+Title: disease ensemble modelling made easy
 Version: 0.0.0.9000
 Authors@R: 
-    person("First", "Last", , "first.last@example.com", role = c("aut", "cre"),
-           comment = c(ORCID = "YOUR-ORCID-ID"))
-Description: What the package does (one paragraph).
-License: `use_mit_license()`, `use_gpl3_license()` or friends to pick a
-    license
+    c(person("Rasmus Skytte", "Randl\u00F8v", , "rske@ssi.dk", role = c("aut", "cre"),
+             comment = c(ORCID = "0000-0002-5860-3838")),
+      person("Marcus Munch", "Gr\u00FCnewald", , "mmgr@ssi.dk", role = c("aut"),
+             comment = c(ORCID = "0009-0006-8090-406X")),
+      person("Lasse Engbo", "Christiansen", , "lsec@ssi.dk", role = c("aut"),
+             comment = c(ORCID = "0000-0001-5019-1931")),
+      person("Kaare", "Gr\u00E6sb\u00F8ll", , "kagr@ssi.dk", role = c("aut"),
+             comment = c(ORCID = "0000-0002-6258-8212")))
+Description: This package facilitates creation of an ensemble of models through modular construction and combinatorics.
+License: GPL-3
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.2
+RoxygenNote: 7.2.3
+Imports:
+    checkmate,
+    glue,
+    purrr,
+    stringr
 Suggests: 
     knitr,
     rmarkdown,
     spelling
 VignetteBuilder: knitr
-URL: https://github.com/RasmusSkytte/diseasy,
```

Comment:
Is there a reason the URL field was removed?

Reply:
It is no longer the valid URL.
Proper URLs for all fields will be added as part of the release issue (when created)

Reply:
Release issue is created now, and the URL is included again